go/obscuronode/obscuroclient/obscuro_client.go

```diff
@@ -36,9 +37,10 @@ type clientImpl struct {
 }
 
 func NewClient(nodeID common.Address, address string) Client {
-	rpcClient, err := rpc.Dial(http + address)
+	httpAddr := http + address
```

Comment:
no need to create a new var for this, can do the join in the args of the message too

Reply:
Fixed.